docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher-launched-kubernetes-clusters-from-backup.md

```diff
@@ -79,23 +79,38 @@ If you are using [local snapshots](./back-up-rancher-launched-kubernetes-cluster
     1. In the **Clusters** page, go to the cluster where you want to remove nodes.
     1. In the **Machines** tab, click **⋮ > Delete** on each node you want to delete. Initially, you will see the nodes hang in a `deleting` state, but once all etcd nodes are deleting, they will be removed together. This is due to the fact that Rancher sees all etcd nodes deleting and proceeds to "short circuit" the etcd safe-removal logic.
 
-1. After all etcd nodes are removed, add a new etcd node that you are planning to restore from.
+1. After all etcd nodes are removed, add a new etcd node that you are planning to restore from. The new node must be assigned the role of `all` (etcd, controlplane, and worker).
```

Comment:
```suggestion
1. After all etcd nodes are removed, add the new etcd node that you are planning to restore from. Assign the new node to the role of `all` (etcd, controlplane, and worker).
```

Reply:
`Assign the new node to the role` - the `to` needs to be dropped otherwise the meaning is changed (and incorrect).

Reply:
Updated the edit suggestion

Reply:
Updated the two others as well

Reply:
Updated with suggestion